docs/tutorials/common-interface-to-reproduce-research-projects.md

```diff
@@ -44,18 +55,76 @@ on Linux, MacOS, Windows and any other platform using this [guide](../installati
 If you encounter any issue, please don't hesitate to tell us via [Discord server](https://discord.gg/JjWNWXKxwT) 
 and/or open a ticket [here](https://github.com/mlcommons/ck/issues).
 
-You also need to install the following CM repository with reusable CM scripts 
+
+
+## First approach: adding CM script to a common repository
+
+
+### Fork main repository
+
+You can simply add a new CM script to the [MLCommons repository](https://github.com/mlcommons/ck) 
+to prepare, run and visualize your experiments without any changes required to your original repository.
+
+First, create your fork of [this repository](https://github.com/ctuning/mlcommons-ck)
+and pull it via CM:
+```bash
+cm pull repo --url={fork of https://github.com/ctuning/mlcommons-ck}
+```
+
+### Copy demo script
+
+You can then copy [the CM script "reproduce-micro-paper-2023-victima"
+from a MICRO paper with Artifact Evaluation](../../cm-mlops/script/reproduce-micro-paper-2023-victima)
+to a new script with your paper/artifact name as follows:
+```bash
+cm copy script reproduce-micro-paper-2023-victima reproduce-micro-paper-2023-{new name}
+```
+
+### Edit wrappers
+
+You can then find its directory and edit `_cm.yaml`, `install_deps.sh`, `run.sh` and `plot.sh`
+to pull your Git repository with your artifacts and call your scripts to install dependencies,
+run experiments and plot results from above wrappers:
+```bash
+cm find reproduce-micro-paper-2023-{new name}
+```
+
+You can now add this directory to your fork and create a PR.
+
+### Set up virtual environment before running experiments
+
+We suggest to use virtual environment installed via CM as follows:
+```bash
+cm run script "install python-venv" --name=ae
+export CM_SCRIPT_EXTRA_CMD="--adr.python.name=ae"
+```
+
+### Run experiments
+
+Each paper should have a CM script with the following variations:
+`install_deps`, `run` and `plot`
+
+It's possible to run them as follows:
+```bash
+cm find script --tags=reproduce,paper
+cm run script {name from above list} --tags=_{variation}
+```
+
+
+
+
+## Second approach: adding CM interface to your research project
+
+First, you also need to install the following CM repository with reusable CM scripts 
 shared by the MLCommons task force on automation and reproducibility
 to make it easier to run and reproduce ML and Systems applications and benchmarks
 across diverse software and hardware:
+
 ```bash
-cm pull repo mlcommons@ck
+cm pull repo ctuning@mlcommons-ck
```

Comment:
Hi @gfursin, is this the expected repository to be used?

Reply:
Yes, because we need to aggregate and test all interfaces in this proxy repository and then create 1 PR to mlcommons@ck ...